### DISCUSSION TITLE
Generate Client SDKs for an API

### DISCUSSION THREAD

**Discussion by pkaruna** [Apicurio/apicurio-registry#2330] [Q&A]
Hello Team,

I would like to know whether apicurio registry supports generation of client sdks for the open api specs hosted in the registry.
I was trying to find out how i can do that, using apicurio registry, but couldn't find any button or tab for doing the same
Can you please help.

Thanks

**Comment by EricWittmann** [comment]
Hi @pkaruna - good question. That feature is in our backlog but not yet implemented.  We've been discussing how best to implement it.  Most likely we'll start out with something simple using e.g. the openapi generator.

(ping @carlesarnal )
